.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,32 @@
+name: "CodeQL"
+
+on:
+ push:
+   branches: [ master ]
+ pull_request:
+   branches: [ master ]
```

Comment:
```suggestion
 push:
   branches:
     - master
 pull_request:
```

Reply:
It's unhappy with this:

<img width="1132" alt="Screenshot 2022-04-18 at 13 26 03" src="https://user-images.githubusercontent.com/355033/163808076-68e40d34-0319-450b-98e0-087576451417.png">

Reply:
Bah, ok, fair enough, add back a `- master` for `pull_request`.

Reply:
Done!